### DISCUSSION TITLE
How to view the result of FMU model?

### DISCUSSION THREAD

**Discussion by suncc** [NTNU-IHB/FMI4j#144] [Q&A]
I have run the demo written as "FMI import", but how can I view the result or how can I get the simulation data?

**Comment by markaren** [comment]
FMI4j is a library, not a tool. That is, you need to write the logic to read the simulation data yourself. That is, FMI4j only exposes the FMI functions for you to use.

**Reply by suncc** [reply]
Can you please give an example on how to read the simulation data?
